packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -90,7 +90,7 @@ export const CollectionsPage = () => {
   }, [searchInput, products, sortOrder]);
 
   return (
-    <>
+    <body>
```

Comment:
This is a big no-no, don't put a `body` tag anywhere inside of your React components since the React app is rendered **into** the `body` and there can only be one body per page.